### PR TITLE
[3.9] Add privacy note to the new captcha plugin

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

Add privacy note to the new captcha plugin

### Testing Instructions

- apply this patch
- notice the privacy notice in the backend

### Expected result

There is also a privacy notice for this plugin the backend

### Actual result

There is NO privacy notice for this plugin the backend

### Documentation Changes Required

None

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_captcha_recaptcha_invisible.ini [hunk 1]
```diff
@@ -24,7 +23,8 @@ PLG_RECAPTCHA_INVISIBLE_EXPIRED_CALLBACK_LABEL="Expired callback"
 PLG_RECAPTCHA_INVISIBLE_EXPIRED_CALLBACK_DESC="(Optional) JavaScript callback, executed when the reCAPTCHA expired"
 PLG_RECAPTCHA_INVISIBLE_ERROR_CALLBACK_LABEL="Error callback"
 PLG_RECAPTCHA_INVISIBLE_ERROR_CALLBACK_DESC="(Optional) JavaScript callback, executed when the reCAPTCHA encounters an error"
-
+; Privacy notice
+PLG_RECAPTCHA_INVISIBLE_PRIVACY_CAPABILITY_IP_ADDRESS="The Invisible reCAPTCHA plugin integrates with Google's reCAPTCHA system as a spam protection service. As part of this service, the IP of the user who is answering the captcha challenge is transmitted to Google."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_captcha_recaptcha_invisible.ini]: Please make it IP address not just IP

**Review by brianteeman — CHANGES_REQUESTED** [review]

**Comment by zero-24** [comment]
fixed here and in the original place.

File: administrator/language/en-GB/en-GB.plg_captcha_recaptcha.ini [hunk 2]
```diff
@@ -58,7 +58,7 @@ PLG_RECAPTCHA_ERROR_VERIFY_PARAMS_INCORRECT="The parameters to verify were incor
 PLG_RECAPTCHA_ERROR_INVALID_REFERRER="reCAPTCHA API keys are tied to a specific domain name for security reasons."
 PLG_RECAPTCHA_ERROR_RECAPTCHA_NOT_REACHABLE="Unable to contact the reCAPTCHA verify server."
 
-PLG_RECAPTCHA_PRIVACY_CAPABILITY_IP_ADDRESS="The reCAPTCHA plugin integrates with Google's reCAPTCHA system as a spam protection service. As part of this service, the IP of the user who is answering the captcha challenge is transmitted to Google."
+PLG_RECAPTCHA_PRIVACY_CAPABILITY_IP_ADDRESS="The reCAPTCHA plugin integrates with Google's reCAPTCHA system as a spam protection service. As part of this service, the IP address of the user who is answering the captcha challenge is transmitted to Google."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_captcha_recaptcha.ini]: Remove `who is` since you have 2 `is`.

**Comment by zero-24** [comment]
pushed.

**Comment by infograf768** [comment]
This is a totally new plugin in J. Please alpha order all the strings.

**Comment by brianteeman** [comment]
@zero-24 thanks for making the change. I missed it in the other file. All good for merging

**Review by brianteeman — APPROVED** [review]
Approve changes

**Comment by zero-24** [comment]
@infograf768 should be alpha ordernd now. please confirm and set RTC so someone can hit the green button ;)

thanks @brianteeman

File: administrator/language/en-GB/en-GB.plg_captcha_recaptcha_invisible.ini [hunk 3]
```diff
@@ -3,30 +3,30 @@
...
-PLG_CAPTCHA_RECAPTCHA_INVISIBLE="CAPTCHA - Invisible reCAPTCHA "
-
 ; Params
-PLG_RECAPTCHA_INVISIBLE_PUBLIC_KEY_LABEL="Site Key"
-PLG_RECAPTCHA_INVISIBLE_PUBLIC_KEY_DESC="Used in the JavaScript code that is served to your users."
-PLG_RECAPTCHA_INVISIBLE_PRIVATE_KEY_LABEL="Secret Key"
-PLG_RECAPTCHA_INVISIBLE_PRIVATE_KEY_DESC="Used in the communication between your server and the reCAPTCHA server. Be sure to keep it a secret."
-PLG_RECAPTCHA_INVISIBLE_BADGE_LABEL="Badge"
-PLG_RECAPTCHA_INVISIBLE_BADGE_DESC="Positioning of the reCAPTCHA badge."
 PLG_RECAPTCHA_INVISIBLE_BADGE_BOTTOMRIGHT="Bottom right"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_captcha_recaptcha_invisible.ini]: Alpha order

File: administrator/language/en-GB/en-GB.plg_captcha_recaptcha_invisible.ini [hunk 4]
```diff
@@ -3,30 +3,30 @@
...
 PLG_RECAPTCHA_INVISIBLE_BADGE_BOTTOMLEFT="Bottom left"
+PLG_RECAPTCHA_INVISIBLE_BADGE_DESC="Positioning of the reCAPTCHA badge."
+PLG_RECAPTCHA_INVISIBLE_BADGE_LABEL="Badge"
 PLG_RECAPTCHA_INVISIBLE_BADGE_INLINE="Inline"
-PLG_RECAPTCHA_INVISIBLE_TABINDEX_LABEL="Tabindex"
-PLG_RECAPTCHA_INVISIBLE_TABINDEX_DESC="The tabindex of the challenge."
 PLG_RECAPTCHA_INVISIBLE_CALLBACK_LABEL="Callback"
 PLG_RECAPTCHA_INVISIBLE_CALLBACK_DESC="(Optional) JavaScript callback, executed after successful reCAPTCHA response"
 PLG_RECAPTCHA_INVISIBLE_EXPIRED_CALLBACK_LABEL="Expired callback"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_captcha_recaptcha_invisible.ini]: Alpha order

File: administrator/language/en-GB/en-GB.plg_captcha_recaptcha_invisible.ini [hunk 5]
```diff
@@ -3,30 +3,30 @@
...
 PLG_RECAPTCHA_INVISIBLE_EXPIRED_CALLBACK_DESC="(Optional) JavaScript callback, executed when the reCAPTCHA expired"
 PLG_RECAPTCHA_INVISIBLE_ERROR_CALLBACK_LABEL="Error callback"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_captcha_recaptcha_invisible.ini]: Alpha order

**Comment by zero-24** [comment]
Do i still miss one @Quy ?

File: administrator/language/en-GB/en-GB.plg_captcha_recaptcha_invisible.ini [hunk 6]
```diff
@@ -3,30 +3,30 @@
...
-PLG_RECAPTCHA_INVISIBLE_TABINDEX_LABEL="Tabindex"
-PLG_RECAPTCHA_INVISIBLE_TABINDEX_DESC="The tabindex of the challenge."
-PLG_RECAPTCHA_INVISIBLE_CALLBACK_LABEL="Callback"
 PLG_RECAPTCHA_INVISIBLE_CALLBACK_DESC="(Optional) JavaScript callback, executed after successful reCAPTCHA response"
-PLG_RECAPTCHA_INVISIBLE_EXPIRED_CALLBACK_LABEL="Expired callback"
+PLG_RECAPTCHA_INVISIBLE_CALLBACK_LABEL="Callback"
 PLG_RECAPTCHA_INVISIBLE_EXPIRED_CALLBACK_DESC="(Optional) JavaScript callback, executed when the reCAPTCHA expired"
-PLG_RECAPTCHA_INVISIBLE_ERROR_CALLBACK_LABEL="Error callback"
+PLG_RECAPTCHA_INVISIBLE_EXPIRED_CALLBACK_LABEL="Expired callback"
 PLG_RECAPTCHA_INVISIBLE_ERROR_CALLBACK_DESC="(Optional) JavaScript callback, executed when the reCAPTCHA encounters an error"
```
Comment (infograf768) [inline administrator/language/en-GB/en-GB.plg_captcha_recaptcha_invisible.ini]: ERROR comes before EXPIRED

File: administrator/language/en-GB/en-GB.plg_captcha_recaptcha_invisible.ini [hunk 7]
```diff
@@ -3,30 +3,30 @@
...
-PLG_RECAPTCHA_INVISIBLE_PRIVATE_KEY_LABEL="Secret Key"
-PLG_RECAPTCHA_INVISIBLE_PRIVATE_KEY_DESC="Used in the communication between your server and the reCAPTCHA server. Be sure to keep it a secret."
-PLG_RECAPTCHA_INVISIBLE_BADGE_LABEL="Badge"
-PLG_RECAPTCHA_INVISIBLE_BADGE_DESC="Positioning of the reCAPTCHA badge."
-PLG_RECAPTCHA_INVISIBLE_BADGE_BOTTOMRIGHT="Bottom right"
 PLG_RECAPTCHA_INVISIBLE_BADGE_BOTTOMLEFT="Bottom left"
+PLG_RECAPTCHA_INVISIBLE_BADGE_BOTTOMRIGHT="Bottom right"
+PLG_RECAPTCHA_INVISIBLE_BADGE_DESC="Positioning of the reCAPTCHA badge."
+PLG_RECAPTCHA_INVISIBLE_BADGE_LABEL="Badge"
 PLG_RECAPTCHA_INVISIBLE_BADGE_INLINE="Inline"
```
Comment (infograf768) [inline administrator/language/en-GB/en-GB.plg_captcha_recaptcha_invisible.ini]: MOve on top of PLG_RECAPTCHA_INVISIBLE_BADGE_LABEL

**Review by infograf768 — CHANGES_REQUESTED** [review]

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on a4ae5e6eea20865a7410c8657d95003a58db3073<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21962">issues.joomla.org/tracker/joomla-cms/21962</a>.</sub>

**Comment by wojsmol** [comment]
@infograf768 Please accept changes on github.

**Review by infograf768 — APPROVED** [review]

**Comment by infograf768** [comment]
@mbabker 
Please merge as some of us are already working on 3.9 translations.
Thanks

**Pull request merged**